src/sentry/models/integrationfeature.py

```diff
@@ -79,11 +79,10 @@ class IntegrationFeature(Model):
     __include_in_export__ = False
 
     # the id of the sentry_app or doc_integration
-    # TODO(CEO): change this to null=False
-    target_id = BoundedBigIntegerField(null=True)
+    target_id = BoundedBigIntegerField(null=False)
```

Comment:
The command line was yelling about his not having a default but I know every existing row has a value, and a default doesn't make sense here since it's an ID of a sentry app or doc integration. Do I need to have one, or is this okay?

Reply:
Should be ok as long as we've verified these IDs not null and are being written for new rows.

Nit: You can just remove `null=False` for both these fields since that's the default